src/app/søknad/omsorgstilbud-step/omsorgstilbud-i-periode/RegistrerOmsorgstilbudPeriode.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { useState } from 'react';
+// import { FormattedMessage } from 'react-intl';
+import { DateRange, InputTime } from '@navikt/sif-common-formik/lib';
+import dayjs from 'dayjs';
+import { Knapp } from 'nav-frontend-knapper';
+import OmsorgstilbudPeriodeDialog from '../../../pre-common/omsorgstilbud-periode/OmsorgstilbudPeriodeDialog';
+import { getDatesInDateRange } from '../../../utils/common/dateRangeUtils';
+import { dateToISODate, ISODateToDate } from '../../../utils/common/isoDateUtils';
+// import useLogSøknadInfo from '../../../hooks/useLogSøknadInfo';
+import { DatoTidMap, TidUkedager } from '../../../types';
+import { OmsorgstilbudPeriodeData } from '../../../pre-common/omsorgstilbud-periode/OmsorgstilbudPeriodeForm';
+
+interface Props {
+    periode: DateRange;
+    onPeriodeChange: (tid: DatoTidMap) => void;
+}
+
+const getTidForUkedag = (tid: TidUkedager, ukedag: number): InputTime | undefined => {
```

Comment:
`getTidForUkedag` og `oppdaterDagerIPeriode` kunne sikkert blitt flyttet til en util, i og med de er like for arbeidstid og omsorgstilbud, men vi kan ta det ved en senere anledning.